library/src/layout/enum.rs

```diff
@@ -36,6 +36,16 @@ use super::GridLayouter;
 /// + Don't forget step two
 /// ```
 ///
+/// You can also use [`enum.item`]($func/enum.item) to programmatically
+/// customize the number of each item in the enumeration:
+/// ```example
```

Comment:
not sure whether the markdown parser will accept this, but better to put a blank line before it